src/main/java/com/gary/backendv2/controller/EnumController.java

```diff
@@ -16,24 +15,24 @@
 @RequestMapping(path = "/enum")
 @RequiredArgsConstructor
 public class EnumController {
+	private static List<String> getEnumValues(Class<? extends Enum> cl) {
+		return Stream.of(cl.getEnumConstants())
+				.map(e -> e.name())
+				.collect(Collectors.toList());
+	}
```

Comment:
`/src/main/java/com/gary/backendv2/utils/EnumUtils`
````suggestion
public class EnumUtils {
    public static List<String> getEnumValues(Class <? extends Enum> enumClass) {
        return Stream.of(cl.getEnumConstants())
				.map(e -> e.name())
				.collect(Collectors.toList());
    }
}
```